service-front/readme.md

```diff
@@ -19,6 +19,12 @@ npm install
 
 ```
 
+## To build grunt for development
+- Apply the scss change
+- Run below command
+
+    ```docker run -it --rm -v $(pwd)/service-front:/srv huli/grunt:alpine grunt build --force```
```

Comment:
Not seen this huli before? Is it trusted?

Reply:
This is what I found from slack archive [from Neil]. I was having issues building grunt locally, but this worked for me on local.

<img width="388" alt="Screenshot 2020-04-20 at 16 02 02" src="https://user-images.githubusercontent.com/6483585/79872519-cc7d5b80-83dd-11ea-85bb-2a45e2cde99c.png">

Reply:
Fair enough :)